scrapy/extensions/feedexport.py

```diff
@@ -156,19 +156,29 @@ def _store_in_thread(self, file):
 
 class FTPFeedStorage(BlockingFeedStorage):
 
-    def __init__(self, uri):
+    def __init__(self, uri, active=False):
         u = urlparse(uri)
         self.host = u.hostname
         self.port = int(u.port or '21')
         self.username = u.username
         self.password = u.password
         self.path = u.path
+        self.active = active
+
+    @classmethod
+    def from_crawler(cls, crawler, uri):
+        return cls(
+            uri=uri,
+            active=crawler.settings['FEED_STORAGE_FTP_ACTIVE'] or False
```

Comment:
Please, use `.getbool()` method as documented at [scrapy.settings.BaseSettings.getbool](https://docs.scrapy.org/en/latest/topics/api.html#scrapy.settings.BaseSettings.getbool)

Reply:
Ok, I was going to use it but since this is the way settings are accessed in this file I decided to follow the pattern.
Will change.